package.json

```diff
@@ -23,35 +22,26 @@
   },
   "homepage": "https://github.com/checkdigit/spectral-config#readme",
   "devDependencies": {
-    "@checkdigit/eslint-config": "7.5.0",
-    "@checkdigit/jest-config": "^2.1.0",
-    "@checkdigit/prettier-config": "^3.2.0",
-    "@checkdigit/typescript-config": "^3.2.0"
+    "@checkdigit/prettier-config": "^4.0.0"
   },
   "peerDependencies": {
-    "@stoplight/spectral-cli": "6.6.0"
+    "@stoplight/spectral-cli": "6.8.0"
```

Comment:
@lukebrdn I assume upgrading to the latest minor version is harmless?

Reply:
Seems so. Nothing in the changelog is jumping out as a potential issue.